src/modules/admin/components/formRules/NewFormRuleDialog.tsx

```diff
@@ -219,9 +244,28 @@ const NewFormRuleDialog: React.FC<Props> = ({
         New rule for: <b>{formTitle}</b>
       </DialogTitle>
       <DialogContent>
-        <Stack gap={2} direction='row' sx={{ mt: 2, mb: 1, display: 'flex' }}>
-          <FormRuleLabelTypography>Applies to</FormRuleLabelTypography>
```

Comment:
this didn't seem accessible since the label isn't associated with the input. Changed to use FormRuleSelectField

Reply:
Hm that's a good point about accessibility. Your concern also applies to the other places in this modal where a pick list's label changes dynamically with another picklist (the other places where we use FormRuleLabelTypography). Would it be possible to update FormRuleSelectField to keep this design, but make it more accessible perhaps by using a dynamic aria label? Or is this pattern just inaccessible by default and there's no saving it? If so, I think we should go back to the design team and ask for revisions.

Designs here https://www.figma.com/board/axGI9vwmuBRC4uZ11ZJ3RP/OP-%2F-HMIS-%2F-5792-%3A-Form-Builder-Workflow?node-id=239-7978&t=f3pwwMGHLq7moNRr-0

Reply:
I played around with it a little and came up with a demo of this idea of the dynamic aria label: https://github.com/greenriver/hmis-frontend/commit/ec0353e4c790e028e6f66a5e85561870167ea600

Sorry it's quite messy and not really functional at the moment, I don't want to put more time into cleaning it up unless you agree that this could be an okay approach!

Reply:
We talked about this briefly at the design session today, design team seemed to lean towards using dynamic aria labels and keeping the design.

Reply:
Hi @martha - thanks for working up a demo! Sorry I missed the last part of the design call. I put a little time on the calendar tomorrow to check in on this if that's okay.

Reply:
We discussed on a call, and decided that the current implementation is fine for an MVP of this task, and we will create a follow-up ticket to improve both the accessibility and design for this form.